website/docs/en/guide/tech/html.mdx

```diff
@@ -0,0 +1,43 @@
+# HTML
+
+Use the following plugins to simplify the creation of HTML files and automatically inject the generated CSS and JavaScript files into the HTML.
+This is especially useful for `Rspack` bundles that include a hash in the filename which changes every compilation.
+
+## HtmlWebpackPlugin
+
+Rspack fully supports [HtmlWebpackPlugin](https://github.com/jantimon/html-webpack-plugin).
+
+```js title="rspack.config.js"
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const path = require('path');
+
+module.exports = {
+  entry: 'index.js',
+  output: {
+    path: path.resolve(__dirname, './dist'),
+    filename: 'index_bundle.js',
+  },
+  plugins: [new HtmlWebpackPlugin()],
+};
+```
+
+For all configuration options, see the [plugin documentation](https://github.com/jantimon/html-webpack-plugin#options).
+
+## HtmlRspackPlugin
```

Comment:
Add `Built-in` to distinguish with the [JS version of HtmlRspackPlugin](https://github.com/rspack-contrib/html-rspack-plugin)

```suggestion
## Built-in HtmlRspackPlugin
```

Reply:
I think we can introduce `HtmlRspackPlugin` before `HtmlWebpackPlugin` because it is recommended.